### PR TITLE
Added FI support.

### PR DESCRIPTION
**Pull request by srfiorini**
Co-Authored-By: Raphael Gomes Santos @raphaelgoms 

Added FI support and some simple tests.

### CODE REVIEW

File: hkpy/hklib/node.py [hunk 1]
```diff
@@ -14,7 +17,7 @@
 from . import HKLink
 from . import HKAnchor
 
-__all__ = ['HKContext', 'HKNode', 'HKReferenceNode', 'HKTrail', 'HKAnyNode']
+__all__ = ['HKContext', 'HKNode', 'HKReferenceNode', 'HKTrail', 'HKAnyNode', 'HKDataNode']
```
Comment (eltonfss) [inline hkpy/hklib/node.py:20]: HKDataNode == HKContentNode?

File: hkpy/hklib/node.py [hunk 2]
```diff
@@ -139,6 +142,66 @@ def to_dict(self) -> Dict:
 
         return jobj
 
+
+class HKDataNode(HKAnyNode):  
+    """
```
Comment (eltonfss) [inline hkpy/hklib/node.py:147]: Would be nice to explain what is an HKDataNode here

File: requirements.txt [hunk 3]
```diff
@@ -6,4 +6,5 @@ urllib3==1.26.5
 pika==1.2.0
 Flask==2.0.1
 flask-cors==3.0.10
-PyJWT==2.3.0
\ No newline at end of file
+PyJWT==2.3.0
+lark==0.11.3
```
Comment (eltonfss) [inline requirements.txt:10]: Just in case someone asks, lark has MIT license: https://www.npmjs.com/package/lark

**Review by eltonfss — APPROVED** [review]
Is tests/data/cat1.png of our authorship? Should we include any sort of license for it?

**Comment by srfiorini** [comment]
> 

I drew `tests/data/cat1.png` myself to avoid those problems. :)

File: hkpy/hklib/node.py (hunk 1 above)
Comment (srfiorini) [reply hkpy/hklib/node.py:20]: done

File: hkpy/hklib/node.py (hunk 2 above)
Comment (srfiorini) [reply hkpy/hklib/node.py]: done

**Pull request merged**